src/gwt/www/index.htm

```diff
@@ -29,6 +29,7 @@
     <link type="text/css" rel="stylesheet" href="css/focus-visible.css" />
     <script type="text/javascript" language="javascript" src="js/inert-polyfill.min.js"></script>
     <script type="text/javascript" language="javascript" src="js/focus-visible.min.js"></script>
+    <script type="text/javascript" language="javascript" src="js/FastestSmallestTextEncoderDecoder.js"></script>
```

Comment:
Unless a script is needed for boot we generally prefer to put it into a clientbundle so that it doesn't require a separate http request.

Reply:
(up to you whether you want to move it!)

Reply:
That's my fault, I put that first polyfill there and set a bad example, both a not putting it in a clientbundle, and putting it in the `<head>`.

Feel free to open an issue to clean that up!